### PR TITLE
Add security:mGetUsers route 

### PR DESCRIPTION
**Pull request by Yoann-Abbes**
## What does this PR do ?

This PR adds a new API route : `security:mGetUsers` in GET and POST
[Jira ticket](https://jira.kaliop.net/browse/KZL-1359)

requested by  https://github.com/Gaiidenn

### How should this be manually tested?

`http://localhost:7512/users/_mGet?ids=test1,test2`

See also that the POST way to hit the route still works.

### CODE REVIEW

File: features-sdk/SecurityController.feature [hunk 1]
```diff
@@ -132,3 +132,15 @@ Feature: Security Controller
...
     And I delete the profile "test-profile"
     And I delete the role "test-role"
+
+  Scenario: Get multiple users
+    Given I create a user "test-user" with content:
+    | profileIds | ["default"] |
+    And I create a user "test-user2" with content:
+    | profileIds | ["default"] |
+    Then I am able to mGet users and get 2 users with the following ids:
+    | ids | ["test-user", "test-user2"] |
```
Comment (Aschen) [inline features-sdk/SecurityController.feature]: Why not pass the ids like this 
```
| "test-user" |
| "test-user2" |
// or
| ["test-user", "test-user2"]
```

File: lib/api/controllers/security.js [hunk 2]
```diff
@@ -705,6 +706,32 @@ class SecurityController extends NativeController {
       });
   }
 
+  /**
+   * Get specific users according to given ids
+   *
+   * @param {Request} request
+   * @returns {Promise<Object>}
```
Comment (Aschen) [inline lib/api/controllers/security.js]:
```suggestion
   * @returns {Promise.<Object>}
```

File: lib/api/controllers/security.js [hunk 3]
```diff
@@ -705,6 +706,32 @@ class SecurityController extends NativeController {
...
+    if ( request.input.body
+      && request.input.body.ids
+      && Object.keys(request.input.body.ids).length
+    ) {
+      ids = this.getBodyArray(request, 'ids');
+    }
+    else {
+      ids = this.getString(request, 'ids').split(',');
+    }
+    return this.kuzzle.repositories.user.loadMultiFromDatabase(ids)
```
Comment (Aschen) [inline lib/api/controllers/security.js]: You should use `async/await`

File: test/api/controllers/security/users.test.js [hunk 4]
```diff
@@ -90,6 +90,17 @@ describe('Test: security controller - users', () => {
     });
   });
 
+  describe('#mGetUsers', () => {
+    it('should throw an error if no ids are given', () => {
```
Comment (Aschen) [inline test/api/controllers/security/users.test.js]: Can you add an unit test on having ids either as an array or a string?

File: features-sdk/step_definitions/security-steps.js [hunk 5]
```diff
@@ -109,3 +110,8 @@ Given('The role {string} should match:', async function (roleId, dataTable) {
     should(actions).match(role.controllers[controller].actions);
   }
 });
+
+Then('I am able to mGet {int} users with the following ids:', async function (count, dataTable) {
```
Comment (Aschen) [inline features-sdk/step_definitions/security-steps.js]: The `count` is unused here. Either check the number of returned user or do not add another variable

File: test/core/models/security/user.test.js [hunk 6]
```diff
@@ -156,4 +156,5 @@ describe('Test: security/userTest', () => {
       InternalError,
       { id: 'security.user.uninitialized' });
   });
+
```
Comment (alexandrebouthinon) [inline test/core/models/security/user.test.js]:
```suggestion
```

File: doc/2/api/controllers/security/m-get-users/index.md [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+
+## Body properties
+
+- `ids`: an array of user kuids to get
+
+---
+
+## Response
+
+Returns a `hits` array of objects. Each object is an user object with the following properties:
```
Comment (scottinet) [inline doc/2/api/controllers/security/m-get-users/index.md]:
```suggestion
Returns a `hits` array of objects. Each object is a user object with the following properties:
```

**Pull request merged**